sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServices.java

```diff
@@ -101,10 +101,6 @@ JobStatistics dryRunQuery(String projectId, JobConfigurationQuery queryConfig, S
     @Nullable
     Table getTable(TableReference tableRef) throws InterruptedException, IOException;
 
-    @Nullable
```

Comment:
Is this deleted a public API?

Reply:
BigQueryServices is a public interface, although it's meant as a private implementation for BigQuery sources and sinks and is not surfaced at the BigQueryIO level. I can restore this functionality and simply stop calling it from BigQueryStorageStreamSource.

Reply:
Done.